spec/support/admin/firm_page.rb

```diff
@@ -7,4 +7,6 @@ class Admin::FirmPage < SitePrism::Page
   element :move_advisers, '.t-move-advisers'
   element :sign_in_as_principal, '.t-sign-in-as-principal'
   element :new_adviser, '.t-new-adviser'
+  element :approved, '.t-approved'
+  element :approve_button, '.t-approve-firm'
```

Comment:
I don't like adding classes to the markup to facilitate testing because it adds unnecessary markup.  I know that the standard on this project has been to use these `t-` classes but if it can be avoided easily, I think we should try to. Is it possible to do something like
```
element :approved_elem, 'input[value="Approve Firm"]'
```

Reply:
will give I a try. Thanks for the suggestion!